### PR TITLE
AMI Meter Data Model

### PR DESCRIPTION
**Pull request by vbayon**
First Commit. Review for structure and integration witn main line.

### CODE REVIEW

**Review by GBBBAS — APPROVED** [review]

**Comment by GBBBAS** [comment]
@vbayon 

For the folder structure, I would suggest:

**AMI Meters**
1. Create a file called `rtdip_sdk/data_models/meters/ami_meters.py`
2. Move the classes `MetaData`, `SeriesType`, `ModelType`, `UoMUsage`, `Usage`, `pjm_voltage_lvl` into `ami_meters.py`. That way it would be simpler to import them in the code such as `from rtdip_sdk.data_models.meters.ami_meters import MetaData` 
3. Move the `utils` folder into the new folder `rtdip_sdk/data_models/meters`

**Weather**

1. Move the `constants/weather_constants.py` file to `rtdip_sdk/data_models/weather` and call the file `constants.py`
 
**Tests**

1. Change the test folders to reflect the above changes

**Transformers**

1. The transformers folder - it would be good to understand the exact function of that code and whether it might be better positioned in the Pipelines section?

** Build Errors**

Can you review the build errors and fix what is being flagged?

**Comment by vbayon** [comment]
@GBBBAS I will look into it!

File: src/sdk/python/rtdip_sdk/data_models/constants/weather_constants.py
Comment (GBBBAS) [inline src/sdk/python/rtdip_sdk/data_models/constants/weather_constants.py:1]: Move this file to weather folder and rename to constants.py

File: src/sdk/python/rtdip_sdk/data_models/series/SeriesType.py
Comment (GBBBAS) [inline src/sdk/python/rtdip_sdk/data_models/series/SeriesType.py:1]: Please see comments [here](https://github.com/rtdip/core/pull/126#issuecomment-1498650663)

File: src/sdk/python/rtdip_sdk/data_models/utils/CreateUsageObject.py
Comment (GBBBAS) [inline src/sdk/python/rtdip_sdk/data_models/utils/CreateUsageObject.py:1]: Please see comments [here](https://github.com/rtdip/core/pull/126#issuecomment-1498650663)

**Review by GBBBAS — CHANGES_REQUESTED** [review]

**Comment by vbayon** [comment]
@GBBBAS I have done a big refactoring. Please check. Thanks! /V

**Review by cching95 — APPROVED** [review]

File: src/sdk/python/rtdip_sdk/data_models/constants/weather_constants.py
Comment (cching95) [reply src/sdk/python/rtdip_sdk/data_models/constants/weather_constants.py:1]: constants file was removed

**Review by cching95 — APPROVED** [review]

**Review by cching95 — APPROVED** [review]

**Pull request merged**